### DISCUSSION TITLE
Where did the community tabs go?

### DISCUSSION THREAD

**Discussion by GitHubinus** [forem/forem#20421] [Q&A]
Hi,

after a recent update of the Forem, some tabs disappeared from the config settings, such as Community Content (Community Name, Tagline, Community Description, Community Member Label).

![Screenshot_1](https://github.com/forem/forem/assets/84869256/38ede6c2-e5d7-4594-8753-ef8bac1f0386)

Is this a bug or was it intended to be this way?

Thanks for the answer.

**Comment by AVSenchenkov** [comment]
F
